testdata/p4_16_samples_outputs/table-entries-exact-ternary-bmv2.p4-stderr

```diff
@@ -0,0 +1,6 @@
+table-entries-exact-ternary-bmv2.p4(65): [--Wwarn=mismatch] warning: P4Runtime requires that Ternary matches have masked-off bits set to 0, updating value 4369 to conform to the P4Runtime specification
```

Comment:
it would be nice if the error message used the same radix as the source code.

Reply:
Do all passes save the base when operating on constants?

Reply:
I won't argue against that if it is straightforward to do, but note that the source code excerpt with the original base is marked with '^^^^' later in the warning message, so that is good.

Reply:
@mbudiu-vmw I made a small change to `IR::Constant::toString` to use the base, so now the values match. The base is not included in the string (e.g. abcd and not 0xabcd), because that's a bigger change.

Reply:
I changed my mind and decided it was also a bit confusing without the base, so I'm amending my commit :)